pkg/render/intrusion_detection.go

```diff
@@ -178,6 +180,10 @@ func (c *intrusionDetectionComponent) Objects() ([]client.Object, []client.Objec
 		c.intrusionDetectionDeployment(),
 	)
 
+	if c.cfg.Tenant != nil {
+		objs = append(objs, c.multiTenantManagedClustersAccess()...)
```

Comment:
We're creating this object even in single-tenant clusters - perhaps the name of this function is wrong?